src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  // static alive = [];
+
+  constructor(name, health = 100,) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    if (this.hidden === false) {
+      this.hidden = true;
+    } else if (this.hidden === true) {
+      this.hidden = false;
```

Comment:
```suggestion
  hide() {
   this.hidden = !this.hiden;
```
try this

Reply:
Yep. Evhenii is right. Let's shorten the code